configure

```diff
@@ -1509,7 +1498,7 @@ fi
 for ac_var in	exec_prefix prefix bindir sbindir libexecdir datarootdir \
 		datadir sysconfdir sharedstatedir localstatedir includedir \
 		oldincludedir docdir infodir htmldir dvidir pdfdir psdir \
-		libdir localedir mandir runstatedir
```

Comment:
You seem to use a patched or modified version of autoconf 2.69. Could you please use an unpatched version to reduce noise? You can use my container image https://quay.io/repository/tiran/cpython_autoconf

Reply:
OK, thanks. Let me have a try.

Reply:
Thanks, your docker image works :)